vnext/Scripts/rnw-dependencies.ps1

```diff
@@ -32,9 +43,9 @@ function InstallVS {
     $channelId = & $vsWhere -version 16 -property channelId
     $productId = & $vsWhere -version 16 -property productId
     $vsInstaller = "$installerPath\vs_installer.exe"
-    $addWorkloads = $vsWorkloads | % { '--add', $_ };
-    Start-Process -PassThru -Wait  -Path $vsInstaller -ArgumentList ("install --channelId $channelId --productId $productId $addWorkloads --quiet" -split ' ')
-
+    $addWorkloads = ($vsWorkloads + $vsComponents) | % { '--add', $_ };
```

Comment:
We already have all of this logic for getting/installing VS components, including getting the installer, in `Install-VSFeatures.ps1` script. It's what the CI loop uses. Can we consolidate this logic?